cloudformation/slack-router-lambda.js

```diff
@@ -0,0 +1,50 @@
+const AWS = require('aws-sdk')
+
+const region = process.env.AWS_REGION
+const accountId = process.env.AWS_ACCOUNT_ID
+
+function parseBody(event) {
+  const eventJSON = JSON.stringify(event.body, null, 2)
+  const eventBody = Buffer.from(eventJSON, 'base64').toString('ascii')
+  const bodyArray = eventBody.split('&')
+
+  const bodyObject = bodyArray.reduce((accumulator, currentValue) => {
+    const [key, value] = currentValue.split('=')
+
+    accumulator[key] = value
+
+    return accumulator
+  }, {})
+
+  return bodyObject
+}
+
+exports.handler = async (event) => {
+  try {
+    const eventBody = parseBody(event)
+    const command = decodeURIComponent(eventBody.command).substring(1)
+
+    const params = {
+      Message: JSON.stringify(eventBody),
+      Subject: 'SNS from Slack Slash Command',
+      TopicArn: `arn:aws:sns:${region}:${accountId}:${command}`,
+    }
+
+    await new AWS.SNS().publish(params).promise()
+
+    return {
+      statusCode: 200,
+      body: 'Your message is being processed...',
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
This is really neat btw, especially for commands that take longer to be processed. Nice work! :raised_hands: